inc/osvr/Util/ClientReportTypesC.h

```diff
@@ -34,11 +34,15 @@
 #include <osvr/Util/Pose3C.h>
 #include <osvr/Util/StdInt.h>
 
+#include <osvr/Util/Vec2C.h>
+#include <osvr/Util/Vec3C.h>
+#include <osvr/Util/ChannelCountC.h>
+
 /* Library/third-party includes */
 /* none */
 
 /* Standard includes */
-/* none */
+#include <stdbool.h>
```

Comment:
We actually have a header for this purpose that also solves the issue of serialization. Something like `<osvr/Util/BoolC.h>`